js/cm-api-v2/PatchSnippet.js

```diff
@@ -0,0 +1,54 @@
+// DocSection: cm_api_v2_patch_snippet
+// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
+// Using ES6 syntax
+import { ManagementClient } from '@kentico/kontent-management';
+
+const client = new ManagementClient({
+  projectId: '<YOUR_PROJECT_ID>',
+  apiKey: '<YOUR_API_KEY>'
+});
+
+client.modifyContentTypeSnippet()
+  .byTypeId('269202ad-1d9d-47\fd-b3e8-bdb05b3e3cf0')
+  //.byTypeCodename('hosted_video')
+  //.byTypeExternalId('snippet-type-123')
+  .withData(
```

Comment:
Is this still valid approach? In [CTC-1022](https://kentico.atlassian.net/browse/CTC-1022), Daniel Pokorny mentioned that `.withData` is deprecated in favor of a builder function. Example can be seen in https://github.com/Kentico/kontent-management-sdk-js/blob/master/test-browser/content-types/add-content-type.spec.ts#L11.

Reply:
Perhaps for some API calls, but not all. I checked the page test-browser page for modifying a snippet, and it looks to use withdata regularly. No sign of builder. https://github.com/Kentico/kontent-management-sdk-js/blob/master/test-browser/content-type-snippets/modify-content-type.spec.ts

I don't think it is possible at this time to taking the formatting used for creating a new content type, and applying it the modify request. The one thing that made me question using builder was that I couldn't find a way to list the actions (separated by commas) without making the return statement thrown an error. I also tried doing just one action, like a replace, but it didn't take.

As for the content type example, we can discuss it outside this pull request with Daniel, and maybe Richard Sustek. If builder is preferred, then the JS SDK read me needs to update, too.

Reply:
If you don't want to approve of this PR yet come Monday, I'll reach out to Richard Sustek to discuss this.

Reply:
It looks like Richard has said in [CTC-1022](https://kentico.atlassian.net/browse/CTC-1022?focusedCommentId=400028) that the withdata parameter is meant for content elements, but is otherwise optional.

Reply:
Let's keep it as it is then. 👍